graphicsmagick/src/wand/magick.rs

```diff
@@ -888,37 +886,33 @@ impl<'a> MagickWand<'a> {
     ///
     /// NAME, VERSION, LIB_VERSION, COPYRIGHT, etc.
     ///
-    pub fn get_configure_info(&mut self, name: &str) -> Result<String, Utf8Error> {
+    pub fn get_configure_info(&mut self, name: &str) -> Option<MagickCString> {
         let name = str_to_c_string(name);
-        let c = unsafe { MagickGetConfigureInfo(self.wand, name.as_ptr()) };
-        c_str_to_string(c)
+        unsafe { MagickCString::new(MagickGetConfigureInfo(self.wand, name.as_ptr())) }
     }
 
     /// <http://www.graphicsmagick.org/wand/magick_wand.html#magickgetcopyright>
     ///
     /// MagickGetCopyright() returns the ImageMagick API copyright as a string.
     ///
-    pub fn get_copyright() -> Result<String, Utf8Error> {
-        let c = unsafe { MagickGetCopyright() };
-        c_str_to_string_no_free(c)
+    pub fn get_copyright() -> &'static CStr {
+        unsafe { CStr::from_ptr(MagickGetCopyright()) }
```

Comment:
@jmjoy It is impossible for these functions, which probably returns a global const variable, to return `NULL` right?

Reply:
I added debug assertion when constructing `&'static CStr` and it passed the CI tests, so maybe it is OK.

Reply:
:ok_hand: